src/platform/plugins/shared/workflows_management/public/widgets/workflow_yaml_editor/ui/workflow_yaml_validation_errors.tsx

```diff
@@ -127,95 +127,99 @@ export function WorkflowYAMLValidationErrors({
+          <EuiFlexGroup direction="column" gutterSize="s">
+            {sortedValidationErrors?.map((error, index) => (
+              <button
+                type="button"
+                key={`${error.startLineNumber}-${error.startColumn}-${error.message}-${index}-${error.severity}`}
+                css={styles.validationError}
+                onClick={() => onErrorClick?.(error)}
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter' || e.key === ' ') {
+                    e.preventDefault();
+                    onErrorClick?.(error);
                   }
-                  size="s"
-                  css={styles.validationErrorIcon}
-                />
-              </EuiFlexItem>
-              <EuiFlexItem>
-                <EuiText color="text" size="xs">
-                  <span>{error.message}</span>
-                </EuiText>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiText color="subdued" size="xs">
-                  <span>{error.source}</span>
-                </EuiText>
-              </EuiFlexItem>
-            </button>
-          ))}
-        </EuiFlexGroup>
-      </div>
-    </EuiAccordion>
+                }}
+                tabIndex={0}
+              >
+                <EuiFlexItem grow={false} css={styles.validationErrorLineNumber}>
+                  <b>{error.startLineNumber}</b>
+                  {':'}
+                  {error.startColumn}
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiIcon
+                    type={
+                      error.severity === 'error'
+                        ? 'errorFilled'
+                        : error.severity === 'warning'
+                        ? 'warningFilled'
+                        : 'iInCircle'
+                    }
+                    color={
+                      error.severity === 'error'
+                        ? 'danger'
+                        : error.severity === 'warning'
+                        ? euiTheme.colors.vis.euiColorVis8
+                        : 'primary'
+                    }
+                    size="s"
+                    css={styles.validationErrorIcon}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiText color="text" size="xs">
+                    <span>{error.message}</span>
+                  </EuiText>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiText color="subdued" size="xs">
+                    <span>{error.source}</span>
+                  </EuiText>
+                </EuiFlexItem>
+              </button>
+            ))}
+          </EuiFlexGroup>
+        </div>
+      </EuiAccordion>
+    </div>
   );
 }
 
 const componentStyles = {
-  accordion: ({ euiTheme }: UseEuiTheme) =>
+  container: ({ euiTheme }: UseEuiTheme) =>
```

Comment:
Since we use `extraAction` now, let's revert styles change here and remove `<div/>` wrapper of accordion.